packages/ckeditor5-dev-utils/lib/translations/multiplelanguagetranslationservice.js

```diff
@@ -211,7 +211,16 @@ module.exports = class MultipleLanguageTranslationService extends EventEmitter {
 			const stringifiedTranslations = JSON.stringify( translatedStrings )
 				.replace( /"([a-z]+)":/g, '$1:' );
 
-			const outputBody = `CKEDITOR_TRANSLATIONS.add('${ language }',${ stringifiedTranslations })`;
+			const outputBody = (
+				// We need to ensure that the CKEDITOR_TRANSLATIONS variable exists and if it exists, we need to extend it.
+				// Use ES5 because this bit will not be transpiled!
+				'(function(d){' +
+					`d['${ language }']=Object.assign(` +
+						`d['${ language }']||{},` +
+						`${ stringifiedTranslations }` +
+					')' +
+				'})(window.CKEDITOR_TRANSLATIONS||(window.CKEDITOR_TRANSLATIONS={}));'
```

Comment:
I think that this kind of access to the global object (through `window`) is going to hurt in some odd envs, but let's keep it for now.